### PR TITLE
LUN-291: Add shadow for share tile on ios

### PR DESCRIPTION
**Pull request by bahaaTuffaha**
This pull request belongs to an issue on our [LUN-291](https://issues.tuerantuer.org/browse/LUN-291).

### CODE REVIEW

File: src/routes/exercise-finished/components/ShareSection.tsx [hunk 1]
```diff
@@ -12,6 +12,9 @@ const Container = styled.View`
   padding: ${props => props.theme.spacings.sm};
   border-radius: 6px;
   shadow-color: ${props => props.theme.colors.shadow};
+  shadow-offset: 0px 6px;
+  shadowopacity: 0.2;
+  shadowradius: 6;
```
Comment (SteffiStuffel) [inline src/routes/exercise-finished/components/ShareSection.tsx:17]: `shadowopacity` and `shadowradius` are no valid properties, need to be `shadow-opacity` and `shadow-radius`.

**Review by SteffiStuffel — COMMENTED** [review]
Tested on android, looks good. Can not test on ios.

**Review by SteffiStuffel — APPROVED** [review]

**Review by steffenkleinle — APPROVED** [review]
Not tested

**Comment by SteffiStuffel** [comment]
@f1sh1918 can you please test this on ios?

**Review by f1sh1918 — APPROVED** [review]
Tested on ios, looks fine :)

**Pull request merged**